README.md

```diff
@@ -54,9 +47,45 @@ async function handleEvent(event) {
 }
 ```
 
+### Options
+
+This function was designed to work with [Worker Sites](TODO:link-marketing-page), but you can customize the behavior of this function by passing the following options into the second argument to `getAssetFromKV`.
+
+#### `ASSET_NAMESPACE`
+
+type: KV Namespace Binding (required if not using Wrangler --site feature)
+
+We assume that your Worker includes a [binding to a Workers KV Namespace](TODO:link-documentation) named `__STATIC_CONTENT`. This binding can be configured by passing the `ASSET_NAMESPACE` option to `getAssetFromKV`. It is further assumed that this namespace consists of static assets such as html, css, javascript, or image files, keyed off of a relative path that roughly matches the assumed url pathname of the incoming request.
+
+#### `__STATIC_CONTENT_MANIFEST` (optional)
```

Comment:
I think we should split off into two sections,  the internal-only (internal to the package) vars and the ones we expect users to use in their index.js.

Reply:
the presentation is these as options; they all have defaults, and this explains what they are, but if folks are using any of them, they need to pass them in at the call site.